azurefunctions-extensions-bindings-blob/pyproject.toml

```diff
@@ -25,7 +25,8 @@ classifiers= [
     ]
 dependencies = [
     'azurefunctions-extensions-base',
-    'azure-storage-blob==12.24.0'
+    'azure-storage-blob==12.24.0',
```

Comment:
Should we use `~=`  so that we don't need to keep updating storage and identity?